vocabularies/Common.xml

```diff
@@ -795,6 +795,15 @@ The minimum structure is
           <Annotation Term="Core.Description" String="Initial value, e.g. empty string, is a valid and significant value" />
         </Property>
       </ComplexType>
+      <ComplexType Name="ValueListParameterConstants" BaseType="Common.ValueListParameter">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Constants" Type="Collection(Edm.PrimitiveType)">
```

Comment:
Do we allow an empty list of constants? If yes, does
```json
"Parameters": [{
  "@type": "#Common.ValueListConstant",
  "ValueListProperty": "Country",
  "Constant": "DE"
}, {
  "@type": "#Common.ValueListConstants",
  "ValueListProperty": "City",
  "Constants": []
}]
```
mean `Country eq 'DE'`?

Reply:
I would say yes - if we should allow it? What do you or the others think?